build/bin/gen-cssfile.js

```diff
@@ -17,7 +17,7 @@ function fileExists(filePath) {
 
 themes.forEach((theme) => {
   var isSCSS = theme !== 'theme-default';
-  var indexContent = '@import "../../element-variables.scss";\n';
+  var indexContent = '@import "./ionicons.min.css";\n@import "../../../element-variables.scss";\n';
```

Comment:
ionicons is never exported in the final css btw for some fonts issues.
So its okay to add it here for testing purposes in the play example